package.json

```
@@ -34,6 +34,7 @@
     "fs-extra": "^10.0.0",
     "isbinaryfile": "^4.0.8",
     "minimist": "^1.2.6",
+    "p-limit": "==3.1.0",
```

```suggestion
    "p-limit": "^3.1.0",
```

issue(blocking): The `install` command in dev is failing due to this syntax being invalid. You'll need to re-run `yarn` as well afterwards to get the lockfile updated.